src/main/kotlin/com/ondosee/global/config/FcmConfig.kt

```diff
@@ -0,0 +1,40 @@
+package com.ondosee.global.config
+
+import com.google.auth.oauth2.GoogleCredentials
+import com.google.firebase.FirebaseApp
+import com.google.firebase.FirebaseOptions
+import com.ondosee.global.config.properties.FcmProperties
+import org.springframework.context.annotation.Configuration
+import java.io.File
+import java.net.URL
+import java.nio.file.Files
+import java.nio.file.Paths
+import javax.annotation.PostConstruct
+
+@Configuration
+class FcmConfig(
+    private val fcmProperties: FcmProperties
+) {
+    companion object {
+        const val PATH = "./credentials.json"
+    }
+
+    @PostConstruct
+    fun init() {
+        runCatching {
+            URL(fcmProperties.url).openStream().use {
+                Files.copy(it, Paths.get(PATH))
+                val file = File(PATH)
+                if (FirebaseApp.getApps().isEmpty()) {
+                    FirebaseOptions.builder()
+                        .setCredentials(GoogleCredentials.fromStream(file.inputStream()))
+                        .build()
+                        .run(FirebaseApp::initializeApp)
+                }
+                file.delete()
+            }
```

Comment:
```suggestion
            URL(fcmProperties.url).openStream().use {
                Files.copy(it, Paths.get(PATH))
            }
            val file = File(PATH)
            if (FirebaseApp.getApps().isEmpty()) {
                FirebaseOptions.builder()
                    .setCredentials(GoogleCredentials.fromStream(file.inputStream()))
                    .build()
                    .run(FirebaseApp::initializeApp)
            }
            file.delete()
```

use를 스트림만 사용하는 부분만 제한해서 사용하는건 어떨까요 ~?!

Reply:
[8ee4966](https://github.com/ondosee/ondosee-Backend/pull/15/commits/8ee4966ba1e582c8a19f7eed0ca77e38b4bbdcf5)

수정했습니다!! 좋은 의견 감사합니다 🙇‍♀️